rust/origen/cli/src/commands/setup.rs

```diff
@@ -64,23 +65,23 @@ pub fn run() {
                 fs::read_to_string(&get_poetry_file).expect("Unable to read Poetry install file");
             let new_data = data.replace(
                 "/bin/env python",
-                &format!("/bin/env {}", PYTHON_CONFIG.command),
+                &format!("/bin/env {}", PYTHON_CONFIG.command.clone().into_string().unwrap()),
             );
             fs::write(&get_poetry_file, new_data).expect("Unable to write Poetry install file");
 
             // Install Poetry
-            os::cmd(&PYTHON_CONFIG.command)
-                .arg(format!("{}", get_poetry_file.display()))
+            Command::new(&PYTHON_CONFIG.command)
+                .arg(get_poetry_file)
                 .arg("--yes")
                 .status()
                 .expect("Something went wrong install Poetry");
 
             if poetry_version().unwrap().major != 1 {
                 // Have to use --preview here to get a 1.0.0 pre version, can only use versions for
                 // official releases
-                os::cmd(&PYTHON_CONFIG.poetry_command)
-                    .arg("self:update")
-                    .arg("--preview")
+                Command::new(&PYTHON_CONFIG.poetry_command)
+                    .arg(OsString::from("self:update"))
+                    .arg(OsString::from("--preview"))
```

Comment:
Is this OsString creation really required here?
I think we have a lot of other places where we don't do this. We should either do this everywhere or nowhere I think.

Reply:
No, I'm not sure that it is needed here. The arg should be internally converting to OsString. I will back out the OsString changes and give it another try.